lib/push-manager.js

```diff
@@ -248,9 +248,10 @@ PushManager.prototype.notifyById = function (installationId, notification, cb) {
 PushManager.prototype.notifyByQuery = function(installationQuery, notification, cb) {
   assert.ok(cb, 'callback should be defined');
   var self = this;
-  var filter = { where: installationQuery };
+  var filter = installationQuery;
```

Comment:
The jsdoc hints installationQuery is the `where` instead of the full `filter`. Do you have the need to use the full `filter`? I can see that pagination options such as `limit` and `offset` are only available with `filter`.